### DISCUSSION TITLE
The installation of Milvus standalone uses Docker. How can I backup it? Can I use the milvus backup tool?

### DISCUSSION THREAD

**Discussion by yesletgo** [milvus-io/milvus#32052] [Q&A and General discussion]
The installation of Milvus standalone uses Docker. How can I backup it? Can I use the milvus backup tool?

**Comment by yhmo** [comment]
milvus-backup tool can backup/restore both for standalone and cluster.

To use milvus-backup:
1. download the latest binary from https://github.com/zilliztech/milvus-backup/releases
2. extract the zip file to a path
3. download this config file: https://github.com/zilliztech/milvus-backup/blob/main/configs/backup.yaml
4. put the backup.yaml into the [path_of_milvus-backup]/configs/backup.yaml
the file structure is like this:
```
├── configs
│   └── backup.yaml
├── milvus-backup
└── README.md
```
5. read the readme to know how to use its commands: https://github.com/zilliztech/milvus-backup?tab=readme-ov-file#command-line
6. make sure the milvus standalone is running, assume there is a collection named "test" in the milvus
7. use this command to back the "test" collection:
`milvus-backup create -c test -n my_backup`

8. use this command to restore the "test" collection:
`milvus-backup restore -c test -s _bak -n my_backup`

**Reply by yesletgo** [reply]
The milvus installation mode is standalone embedded. 


I installed it through the following methods:
Wget https://raw.githubusercontent.com/milvus-io/milvus/master/scripts/standalone_embed.sh
Bash standalone embedded. sh start

All are default configurations, so how does mini connect?
Mini:
Address: localhost # Address of MinIO/S3
Port: 9000 # Port of MinIO/S3
AccessKeyID: minioadmin # accessKeyID of MinIO/S3
SecretAccess Key: minioadmin # MinIO/S3 encryption string
UseSSL: false # Access to MinIO/S3 with SSL
BucketName: a-bucket # Bucket name in MinIO/S3
RootPath: files # The root path where the message is stored in MinIO/S3


I also added port mapping, but I couldn't connect.
[error="Get \" appears http://localhost:9000/a -Bucket/? Location=\“
![image](https://github.com/milvus-io/milvus/assets/21998052/c8469db1-ddfe-48b2-81c5-ba6c959de995)

**If I just want to use the standalone embedded mode, how do I backup it？**
